src/pages/book/publish.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+// Components
+import BookCard from "../../components/book/bookCard";
+import PostBook from "../../components/book/PostBook";
+// Util
+import ScreamSkeleton from "../../util/ScreamSkeleton";
+// MUI Core
+import Grid from "@material-ui/core/Grid";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogActions from "@material-ui/core/DialogActions";
+
+// Redux
+import { connect } from "react-redux";
+// import { getScreams } from "../../redux/actions/dataActions";
+import { getBookData } from "../../redux/actions/dataActions";
+import axios from "axios";
+
+const styles = (theme) => ({
+  ...theme.spread,
+  inProgress: {
+    marginBottom: "5em",
+  },
+  inProgressTitle: {
+    color: "#1c2a48",
+    fontWeight: "bolder",
+    fontFamily: "Times New Roman",
+    textShadow: "2px 2px #fff",
+  },
+  bookCardContainer: {
+    display: "flex",
+  },
+  publishHeader: {
+    background: "red",
+    width: "100%",
+  },
+  publishBody: {
+    display: "flex",
+  },
+});
+class bookreview extends Component {
```

Comment:
in general, I think component name should match name of file if its a general export